hangman/lib/hangman/server.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Hangman.Server do
```

Comment:
# -10

In the course, we discussed splitting this functionality up into the API side and the code that runs in the server.